tsconfig.json

```diff
@@ -12,5 +12,6 @@
 	},
 	"typedocOptions": {
 		"out": "build/typedoc"
-	}
+	},
+	"include": ["src/**/*", "test/**/*"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected object keys to be in natural insensitive ascending order. 'include' should be before 'typedocOptions'.](https://app.codacy.com/gh/cpuabuse/cross-cat/pullRequest?prid=5222203)